content/updates/2021-03/3-Attribute-guidelines.md

```diff
@@ -0,0 +1,23 @@
+---
+pim_announcement_alt_img: "An example of attribute guidelines"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Display attribute guidelines
```

Comment:
```suggestion
# Add descriptions to your attributes
```

Reply:
The name of the feature is "Attribute guidelines". I feel like we should at least keep its name in the title.

Reply:
I'd rather have a clearer title, and then introduce the feature and its name.
People don't know what are Attribute Guidelines.

Reply:
Ok, I understand. In this case, I think "guidelines" is more closed to reality than "description".
```suggestion
# Display guidelines on your attributes
```

Reply:
Sounds good.